src/docs/content/observability/oci-healthcheck.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "OCI Health Check"
```

Comment:
Use the full (exact) service name

Reply:
It's OCI Health Checks. Corrected.
https://docs.cloud.oracle.com/en-us/iaas/Content/HealthChecks/Concepts/healthchecks.htm
I didnt expand OCI as it wont fit the side bar.